### PR TITLE
BREAKING CHANGE: major version bump to v6

### PR DESCRIPTION
**Pull request by reesscot**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by calebpollman — APPROVED** [review]

**Review by thaddmt — DISMISSED** [review]

File: examples/next/package.json [hunk 1]
```diff
@@ -11,11 +11,11 @@
   },
   "dependencies": {
     "@aws-amplify/geo": "^3.0.1",
-    "@aws-amplify/ui-react": "^5.3.2",
-    "@aws-amplify/ui-react-geo": "^1.0.2",
+    "@aws-amplify/ui-react": "^6.0.0",
+    "@aws-amplify/ui-react-geo": "^2.0.0",
     "@aws-amplify/ui-react-liveness": "^2.0.11",
```
Comment (reesscot) [inline examples/next/package.json:16]: left this since we didn't bump liveness

**Review by esauerbo — DISMISSED** [review]

File: packages/vue/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # @aws-amplify/ui-vue
 
+## 4.0.0
+
+### Major Changes
+
+- Major version bump for all Amplify UI packages due to uprade of peerDependency aws-amplify to v6
```
Comment (reesscot) [inline packages/vue/CHANGELOG.md:7]:
```suggestion
- Major version bump for all Amplify UI packages due to upgrade of peerDependency aws-amplify to v6
```

File: packages/angular/projects/ui-angular/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
 # @aws-amplify/ui-angular
 
+## 5.0.0
+
+### Major Changes
+
+- Major version bump for all Amplify UI packages due to uprade of peerDependency aws-amplify to v6
```
Comment (esauerbo) [inline packages/angular/projects/ui-angular/CHANGELOG.md:7]:
```suggestion
Major version bump for all Amplify UI packages due to upgrade of peerDependency aws-amplify to v6
```

File: packages/react-auth/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# @aws-amplify/ui-react-auth
```
Comment (dbanksdesign) [inline packages/react-auth/CHANGELOG.md:1]: Do we need this?

**Review by esauerbo — APPROVED** [review]

**Review by calebpollman — DISMISSED** [review]
🚢

**Review by dbanksdesign — APPROVED** [review]
LFG

File: packages/react-auth/CHANGELOG.md (hunk 4 above)
Comment (calebpollman) [reply packages/react-auth/CHANGELOG.md:1]:
```suggestion
```

File: package.json [hunk 5]
```diff
@@ -122,5 +122,6 @@
     "turbo": "^1.2.8",
     "typescript": "^5.2.2",
     "wait-on": "^6.0.0"
-  }
+  },
+  "version": "0.0.0"
```
[the remark this answers is not in the document]
Comment (reesscot) [reply package.json]: fixed

**Review by calebpollman — APPROVED** [review]

**Review by dbanksdesign — APPROVED** [review]

**Pull request merged**